Makefile

```
@@ -26,3 +26,16 @@ purge:                      ## Clean cache and leftovers. Please run this when s
 	git reset --hard && git clean -xdff
 	git submodule update
 	git submodule foreach 'git reset --hard && git clean -xdff'
+
+fb:                         ## Creates feature build branch.
+  # Usage: make fb mainBranch=PMM-2.0 featureBranch=PMM-XXXX-name submodules="pmm pmm-managed"
+	git checkout $(mainBranch)
```

We can set PMM-2.0 as default value for main branch
```suggestion
	$(eval MAIN_BRANCH = $(or $(mainBranch),PMM-2.0))
	git checkout $(MAIN_BRANCH)
```

---

The reason I didn't do it at first is because one day we might change our branch namings and forget to update this. But maybe was just overthinking it. Thanks for the suggestions @artemgavrilov, done!